liboptv/include/epi.h

```diff
@@ -0,0 +1,44 @@
+#ifndef EPI_H
+#define EPI_H
+
+#define MIN(a,b) (((a)<(b))?(a):(b))
+#define MAX(a,b) (((a)>(b))?(a):(b))
+#define MAXCAND 200 /* see typedefs.h for the reference */
+
+
+#include "calibration.h"
+#include "tracking_frame_buf.h"
+#include "parameters.h"
+#include "lsqadj.h"
+#include "ray_tracing.h"
+#include "multimed.h"
+#include "vec_utils.h"
+#include "imgcoord.h"
+
+
+typedef struct {
+  int  	pnr;
+  double  tol, corr;
+} candidate;
+
+typedef struct
+{
+  int pnr;
+  double x, y;
+}
+coord_2d;
+	
+
+void epi_mm (double xl, double yl, Calibration *cal1,
```

Comment:
Yes please. I'm still looking at the indexing thing (also took a short break for my experiment).